enhancements/console/customize-catalogs.md

```diff
@@ -0,0 +1,172 @@
+---
+title: customize-catalogs
+authors:
+  - "@lokanandaprabhu"
+reviewers:
+  - "@jerolimov"
+  - "@spadgett"
+approvers:
+  - "@spadgett"
+  - "@cvogt"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2022-07-27
+last-updated: 2022-08-25
+tracking-link: 
+  - https://issues.redhat.com/browse/ODC-6695
+status: implementable
+---
+
+# Customize Catalogs
+
+## Summary
+
+The OpenShift Console provides a "Developer Catalog" which enables 
+the user to create apps and services based on Operator backed services, helm charts, templates, etc.
+
+Cluster admins can provide additional catalog items and types by installing operators or adding helm repositories.
+Cluster administrators want to hide these types.
+
+Under the developer catalog,
+there are categories and sub-categories. When users select a sub-category, they can see the list
+of types to install. This enhancement allows cluster admins to selectively disable these types.
+
+![Developer catalog and types list](customize-catalogs.png)
+ 
+## Motivation
+
+Cluster administrators don't want the users to use the catalog 
+to install anything in the cluster until
+customization. Cluster administrators want to 
+disable developer catalog or certain types for all users.
```

Comment:
This sounds like the developer catalog doesn't make sense until its customized. I would rephrase it a little bit so that its clear that the developer catalog is fine for most people, but this allows the admin to disable some unwanted (unsupported by their own support team) catalogs -- without uninstalling an operator that extends this or disabling for example Helm completely.

```suggestion
The developer catalog can be extended by plugins and also dynamically by installing a console plugin or an operator.

Some customers want to align their development process and have teams that support other development teams with their work.
For this cluster administrators want to hide developer sub-catalog (types) without disabling a full console plugin, for example by uninstalling an operator.
By hiding for example "Service Backed operators", "Helm", etc. they can still install them via the CLI, see their resources in the topology, but don't show them in the add from developer catalog flow.
```